src/patternfly/components/Box/styles.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../patternfly-utilities";
+
+:root {
+  // Component variables
+  --pf-c-box--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-box--BoxShadow: var(--pf-global--BoxShadow--sm);
+
+  // Box Header variables
+  --pf-c-box__header--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-box__header--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-box__header--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-box__header--PaddingLeft: var(--pf-global--spacer--xl);
+
+  // Box Body variables
+  --pf-c-box__body--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-box__body--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-box__body--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-box__body--PaddingTop: var(--pf-global--spacer--sm);
+
+  // Box Footer variables
+  --pf-c-box__footer--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-box__footer--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-box__footer--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-box__footer--PaddingLeft: var(--pf-global--spacer--xl);
+
+}
+
+.pf-c-box {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
```

Comment:
why do we set the height of the box?

Reply:
If it's not set, then the box did not stretch (the body area) to fill the vertical space given, causing the footer to float up. If that's not the desired behavior, I can remove it.

Reply:
When we place the box in a layout like grid or gallery, is the expectation that the box fill the height of it's container? Are there cases or layouts where that shouldn't happen?

Reply:
(@srambach's comment wasn't visible when I posted mine—I was replying to @andresgalante's original comment)